src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMHeadDiscoveryHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.atlassian.bitbucket.jenkins.internal.scm;
+
+import jenkins.scm.api.SCMHead;
+import jenkins.scm.api.SCMRevision;
+
+import java.util.stream.Stream;
+
+/**
+ * TODO: Update doc with relevant methods when BitbucketSCMSource is implemented.
```

Comment:
how will we track this to ensure it is done? Are we relying on reviewers of the branch->master PR to catch?

Reply:
I have created a ticket to track this once we start work on SCMSource